command/agent/cache_end_to_end_test.go

```diff
@@ -176,10 +175,10 @@ func testhelperAutoAuth(t *testing.T, fileSinks []*sink.SinkConfig, tokenFile st
 	cluster.Start()
 
 	vault.TestWaitActive(t, cluster.Cores[0].Core)
-	client = cluster.Cores[0].Client
+	client := cluster.Cores[0].Client
 
-	defer os.Setenv(api.EnvVaultAddress, os.Getenv(api.EnvVaultAddress))
-	os.Setenv(api.EnvVaultAddress, client.Address())
+	defer os.Setenv(api.EnvVaultAgentAddr, os.Getenv(api.EnvVaultAgentAddr))
```

Comment:
Why is this being set to `EnvVaultAgentAddr`?

Reply:
I was originally thinking of using that to trigger the new agent-specific client behaviour, but then remembered I could do so by instead setting Config.AgentAddress, which I'm already doing.  Will revert this change, thanks.